integration-cli/daemon/daemon.go

```diff
@@ -180,6 +180,7 @@ func (d *Daemon) getClientConfig() (*clientConfig, error) {
 	if err := sockets.ConfigureTransport(transport, proto, addr); err != nil {
 		return nil, err
 	}
+	transport.DisableKeepAlives = true
```

Comment:
Or have an option to pass additional parameters to `ConfigureTransport`. Looks like that function is to "configure" the transport, but now only does the job "half".

Likely out of scope for this PR, but we may want to look into that